rust/template/src/flatbuf.rs

```diff
@@ -464,26 +466,29 @@ impl<'a> FromFlatBuffer<fb::__Command<'a>> for DDValueUpdate {
         let val_table = cmd.val().ok_or_else(|| {
             format!("Update::from_flatbuf: invalid buffer: failed to extract value")
         })?;
-        match cmd.operation() {
-            Operation_InsertOrDelete => {
+        match FBOperation::from_i8(cmd.operation()) {
+            Some(FBOperation::InsertOrDelete) => {
                 let val = relval_from_flatbuf(relid, val_table)?;
                 match cmd.weight() {
                     1 => Ok(DDValueUpdate(Update::Insert { relid, v: val })),
                     (-1) => Ok(DDValueUpdate(Update::DeleteValue { relid, v: val })),
                     w => Err(format!("Update::from_flatbuf: non-unit weight {}", w)),
                 }
             }
-            Operation_InsertOrUpdate => {
+            Some(FBOperation::InsertOrUpdate) => {
                 let val = relval_from_flatbuf(relid, val_table)?;
                 Ok(DDValueUpdate(Update::InsertOrUpdate { relid, v: val }))
             }
-            Operation_DeleteByKey => {
+            Some(FBOperation::DeleteByKey) => {
                 let val = relkey_from_flatbuf(relid, val_table)?;
                 Ok(DDValueUpdate(Update::DeleteKey { relid, k: val }))
             }
-            o => Err(format!(
+            Some(o) => Err(format!(
```

Comment:
You cover all cases, so this should be unreachable. We should be able to remove this case in the next PR.